src/dft.cpp

```diff
@@ -316,21 +316,21 @@ double dft_chunk::norm2() const {
   return sum;
 }
 
-// return the minimum decimation factor across
+// return the maximum decimation factor across
 // all dft regions
-int fields::min_decimation() const {
-  int mindec = std::numeric_limits<int>::max();
+int fields::max_decimation() const {
+  int maxdec = std::numeric_limits<int>::min();
```

Comment:
```suggestion
  int maxdec = 1;
```
We never want to return a number less than 1 here.

Reply:
Fixed.